Department-for-Transport-Disruptions/create-disruptions-data
# feat: Setup initial SIRI-SX Generator Lambda

## Description

Node.js lambda (written in TypeScript) created which parses and prints the contents of an S3 bucket after a file is uploaded to the bucket

## Testing instructions

Upload a file to s3 (cdd-disruptions-json) and check logs to see if the lambda (cddsirigenerator) was triggered and the contents should print on your terminal
